ios-bills-and-subscriptions/Pave/Services/NetworkService.swift

```diff
@@ -22,12 +22,12 @@ class NetworkService {
 	}
 	
 	var token: String {
-		return <#insert token here#>
+		return "tn4wIX5U2fBNU9A1romAL4vFVo64TiSB"
```

Comment:
Hmm, this is now exposed. You might want to rotate this key before merging. Also, you may want to change this back to `<#insert token here#>`